runtime/vm/vmthread.c

```diff
@@ -598,10 +605,17 @@ threadParseArguments(J9JavaVM *vm, char *optArg)
 	omrthread_lib_clear_flags(J9THREAD_LIB_FLAG_SECONDARY_SPIN_OBJECT_MONITORS_ENABLED | J9THREAD_LIB_FLAG_FAST_NOTIFY);
 #if defined(OMR_THR_SPIN_WAKE_CONTROL)
  	if (cpus < OMRTHREAD_MINIMUM_SPIN_THREADS) {
- 		**(UDATA**)omrthread_global("maxSpinThreads") = OMRTHREAD_MINIMUM_SPIN_THREADS;
+ 		maxSpinThreadsLocal = OMRTHREAD_MINIMUM_SPIN_THREADS;
```

Comment:
Can you introduce a scope here so the `UDATA maxSpinThreadsLocal = 0;` declaration can occur here and avoid the other #ifdef block?

Reply:
added scope and removed #ifdef block.